src/main/java/com/nextstep/lotto/domain/WinningLotto.java

```diff
@@ -6,12 +6,12 @@
     private final Lotto lotto;
     private final LottoNumber bonusNumber;
 
-    public WinningLotto(List<Integer> numbers, int bonusNumber) {
-        if (numbers.contains(bonusNumber)) {
-            throw new IllegalArgumentException("numbers cannot conatins bonus number");
+    public WinningLotto(List<LottoNumber> numbers, int bonusNumber) {
```

Comment:
```suggestion
    public WinningLotto(Lotto lotto, int bonusNumber) {
```
일급 컬렉션을 활용해보면 좋겠네요
List<LottoNumber>로 받게 되면 로또 번호 10개를 넘겨도 통과가 될테니 중복된 유효성 검증 로직이 또 생기겠죠
getMatchedCount, checkBonusNumber 메서드도 변경해주세요

Reply:
어차피 Lotto로 만들면서 통과가 안돼서 유효성 검증 로직이 중복되진 않겠지만.. 말씀하신대로 기껏 Lotto로 감싸놓고 List로 받고 있는게 좀 별로네요 😓  이게 다 Lotto에 getter 안없애서 .. 의견 감사합니다 😃